### PR TITLE
 Remove title page from documentation

### LINKED ISSUE #151: Remove title page from documentation
The landing page should be the Overview.

### PR DESCRIPTION
**Pull request by AnkushinDaniil**
Fixes #151

## Summary by CodeRabbit

- **New Features**
	- Introduced automation for building the Docusaurus website upon pull requests, improving review efficiency.
	- Updated documentation routing for a streamlined access point.

- **Bug Fixes**
	- Changed the handling of broken links from strict errors to warnings, enhancing user experience during builds.

- **Chores**
	- Improved clarity and accessibility of documentation links for better navigation.
	- Simplified footer content for enhanced user experience.
	- Modified document metadata to enhance routing and navigation in the documentation.

### CODE REVIEW

**Comment by AnkushinDaniil** [comment]
@joshklop 
review request

File: doc/docusaurus.config.ts [hunk 1]
```diff
@@ -18,7 +18,8 @@ const config: Config = {
   organizationName: 'polymerdao', // Usually your GitHub org/user name.
   projectName: 'monomer', // Usually your repo name.
 
-  onBrokenLinks: 'throw',
+  // TODO: Change onBrokenLinks to "throw" when possible
+  onBrokenLinks: 'warn',
```
Comment (joshklop) [inline doc/docusaurus.config.ts:22]: why is this necessary?

Comment (AnkushinDaniil) [reply doc/docusaurus.config.ts:22]: rebase issue

File: doc/docs/intro.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 1
+slug: /
```
Comment (joshklop) [inline doc/docs/intro.md:2]: The overview should still be first on the sidebar, right now it displays as the second item.
image

**Review by joshklop — CHANGES_REQUESTED** [review]

File: doc/docs/intro.md [hunk 3]
```diff
@@ -12,11 +13,11 @@ Monomer is pre-production software and should be used with caution.
...
+
 From the [OP stack](https://specs.optimism.io/protocol/overview.html#components)'s perspective, Monomer replaces the default Ethereum
 compatible execution engine. From the [Cosmos application](https://docs.cosmos.network/v0.50/learn/intro/why-app-specific#what-are-application-specific-blockchains)'s perspective,
 Monomer replaces the CometBFT consensus layer.
 
-:::note[Architecture] 
+:::note[Architecture]
 ![Architecture](/img/architecture.png)
 :::
-
```
Comment (joshklop) [inline doc/docs/intro.md:22]: Unrelated changes, it appears?

File: doc/docs/intro.md [hunk 4]
```diff
@@ -19,4 +20,3 @@ Monomer replaces the CometBFT consensus layer.
 :::note[Architecture] 
 ![Architecture](/img/architecture.png)
 :::
-
```
Comment (joshklop) [inline doc/docs/intro.md:22]: unrelated change?

**Review by joshklop — CHANGES_REQUESTED** [review]

**Pull request merged**